CategoryMicro/src/test/java/com/grizzly/categorymicro/CategoryMicroApplicationTests.java

```diff
@@ -21,13 +21,67 @@
 	@Autowired
 	CategoryService testService;
 
-	@InjectMocks
-	CategoryService mockService;
+	//@InjectMocks
```

Comment:
These tests need to be uncommented. Also, all of the tests related to GrizLibrary can be taken out, as they are being run in the library itself.